src/OrbitGl/App.cpp

```diff
@@ -345,73 +346,73 @@ void OrbitApp::OnCaptureStarted(const orbit_grpc_protos::CaptureStarted& capture
+        capture_started_callback_(file_path);
 
-    if (!capture_data_->instrumented_functions().empty()) {
-      ORBIT_CHECK(select_live_tab_callback_);
-      select_live_tab_callback_();
-    }
+        if (!GetCaptureData().instrumented_functions().empty()) {
+          ORBIT_CHECK(select_live_tab_callback_);
+          select_live_tab_callback_();
+        }
 
-    FireRefreshCallbacks();
+        FireRefreshCallbacks();
 
-    main_window_->AppendToCaptureLog(
-        MainWindowInterface::CaptureLogSeverity::kInfo, absl::ZeroDuration(),
-        absl::StrFormat(
-            "Capture started on %s.",
-            absl::FormatTime(absl::FromUnixNanos(capture_started.capture_start_unix_time_ns()),
-                             absl::LocalTimeZone())));
-
-    orbit_version::Version capture_version{capture_started.orbit_version_major(),
-                                           capture_started.orbit_version_minor()};
-    orbit_version::Version current_version = orbit_version::GetVersion();
-    if (capture_version > current_version) {
-      std::string warning_message = absl::Substitute(
-          "The capture was taken with Orbit version $0.$1, which is higher than the "
-          "current version. Please open the capture using Orbit v$0.$1 or above.",
-          capture_version.major_version, capture_version.minor_version);
-      main_window_->AppendToCaptureLog(MainWindowInterface::CaptureLogSeverity::kSevereWarning,
-                                       absl::ZeroDuration(), warning_message);
-      SendWarningToUi("Capture", warning_message);
-    }
-    absl::MutexLock lock(&mutex);
-    initialization_complete = true;
-  });
+        main_window_->AppendToCaptureLog(
+            MainWindowInterface::CaptureLogSeverity::kInfo, absl::ZeroDuration(),
+            absl::StrFormat(
+                "Capture started on %s.",
+                absl::FormatTime(absl::FromUnixNanos(capture_started.capture_start_unix_time_ns()),
+                                 absl::LocalTimeZone())));
+
+        orbit_version::Version capture_version{capture_started.orbit_version_major(),
+                                               capture_started.orbit_version_minor()};
+        orbit_version::Version current_version = orbit_version::GetVersion();
+        if (capture_version > current_version) {
+          std::string warning_message = absl::Substitute(
+              "The capture was taken with Orbit version $0.$1, which is higher than the "
+              "current version. Please open the capture using Orbit v$0.$1 or above.",
+              capture_version.major_version, capture_version.minor_version);
+          main_window_->AppendToCaptureLog(MainWindowInterface::CaptureLogSeverity::kSevereWarning,
+                                           absl::ZeroDuration(), warning_message);
+          SendWarningToUi("Capture", warning_message);
+        }
+        absl::MutexLock lock(&mutex);
+        initialization_complete = true;
+      });
```

Comment:
Can you point out what changed here? It looks like a formatting only change. (If that's the case, could you try to keep them out of your PR in the future and make a separate one, since otherwise reviewers spent a lot of time trying to find the difference?)

Reply:
I think, it's using of capture_data_ getters instead (as opposed to direct access to the field) caused such drastic changes to formatting. How should've I split this?